scripts/sort_contibutors/main.py

```diff
@@ -0,0 +1,217 @@
+import requests
+import os
+from dotenv import load_dotenv
+
+# This function parses the contribution list, sorting
+# the users per its ranks
+
+
+def print_file(s: str, flag: bool) -> None:
+
+    # True for MD , false for HTML file
+    if flag:
+        f = open('contributors_file.md', 'w')
+        f.write(s)
+        return
+    f = open('contributors_file.html', 'w')
+    f.write(s)
+
+
+def print_md(user_list: dict, label="") -> str:
+
+    string = '{}\n'.format(label)
+    for value in user_list:
+        string += '- [{} @{}](https://www.github.com/{})\n'.format(value['name'],
+                                                                   value['username'], value['username'])
+    return string + '\n'
+
+
+def print_html(leaders: dict, top_contributors: dict, contributors: dict, testers: dict, special_thanks: dict) -> str:
+
+    string = '<html><head></head><body>\n'
+
+    string += '<h1>Leaders</h1>\n'
+    string += '<ul>\n'
+    for value in leaders:
+        string += '<li><a href=\'https://www.github.com/{}\'>{} @{}</a></li>\n'.format(
+            value['username'], value['name'], value['username'])
+    string += '</ul>\n'
+
+    string += '\n<h1>Top contributors</h1>\n'
+    string += '<ul>\n'
+    for value in top_contributors:
+        string += '<li><a href=\'https://www.github.com/{}\'>{} @{}</a></li>\n'.format(
+            value['username'], value['name'], value['username'])
+    string += '</ul>\n'
+
+    string += '\n<h1>Contributors</h1>\n'
+    string += '<ul>\n'
+    for value in contributors:
+        string += '<li><a href=\'https://www.github.com/{}\'>{} @{}</a></li>\n'.format(
+            value['username'], value['name'], value['username'])
+    string += '</ul>\n'
+
+    string += '<h1>Testers</h1>\n'
+    string += '<ul>\n'
+    for value in testers:
+        string += '<li><a href=\'https://www.github.com/{}\'>{} @{}</a></li>'.format(
+            value['username'], value['name'], value['username'])
+    string += '</ul>\n\n'
+
+    string += '<h1>Special thanks</h1>\n'
+    string += '<ul>\n'
+    for value in special_thanks:
+        string += '<li><a href=\'https://www.github.com/{}\'>{} @{}</a></li>\n'.format(
+            value['username'], value['name'], value['username'])
+    string += '</ul>\n'
+
+    string += '</body><html>\n'
+    return string
+
+
+def parse_contributor_list(user_list: list, user_token: str) -> list:
+    contributors = []
+
+    for element in user_list:
+        ranking = element['contributions']
+        username = element['login']
+        name = get_fullname(username, user_token)
+
+        black_list = ['DerGut', 'bkimminich', 'MichaelEischer', 'rseedorff', 'jonasbg', 'scornelissen85', 'zadjadr', 'stuebingerb', 'sydseter', 'troygerber', 'skandix', 'saymolet',
```

Comment:
can we rename `black_list` to `leaders_and_multijuicer`
```suggestion
        leaders_and_multijuicer = ['DerGut', 'bkimminich', 'MichaelEischer', 'rseedorff', 'jonasbg', 'scornelissen85', 'zadjadr', 'stuebingerb', 'sydseter', 'troygerber', 'skandix', 'saymolet',
```